ui/src/components/logs/LogLine.vue

```diff
@@ -159,14 +169,17 @@
         white-space: pre-wrap;
         word-break: break-all;
         display: flex;
-        align-items: baseline;
+        align-items: center;
```

Comment:
**This is to center the icon. It was observed on baseline when logs were in two or more lines.**